README.org

```diff
@@ -344,6 +344,44 @@
       }
     #+end_src
 
+    or use the provided ~homeManagerModules.impermanence~ flake output:
+
+    #+begin_src nix
+      {
+        inputs = {
+          home-manager.url = "github:nix-community/home-manager";
+          impermanence.url = "github:nix-community/impermanence";
+        };
+
+        outputs =
+          {
+            home-manager,
+            nixpkgs,
+            impermanence,
+            ...
+          }:
+          {
+            nixosConfigurations.sythe = nixpkgs.lib.nixosSystem {
+              system = "x86_64-linux";
+              modules = [
+                {
+                  imports = [ home-manager.nixosModules.home-manager ];
```

Comment:
Is this correct, regarding https://github.com/nix-community/impermanence/pull/181/files#diff-ecbc1aa90e9ff97a00b0b2aab1551bceee0c4d21993146bdcb1af4de31c9cac6R335?